Userland/Services/DHCPClient/main.cpp

```diff
@@ -9,15 +9,22 @@
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
-ErrorOr<int> serenity_main(Main::Arguments)
+ErrorOr<int> serenity_main(Main::Arguments args)
 {
+    if (args.argc < 2) {
+        dbgln("Not running with no interfaces enabled");
+        return 0;
+    }
     TRY(Core::System::pledge("stdio unix inet cpath rpath"));
     Core::EventLoop event_loop;
 
     TRY(Core::System::unveil("/proc/net/", "r"));
     TRY(Core::System::unveil(nullptr, nullptr));
 
-    auto client = TRY(DHCPv4Client::try_create());
+    Vector<String> interfaces;
+    for (size_t i = 1; i < args.strings.size(); i++)
+        interfaces.append(args.strings[i]);
```

Comment:
Can we use `ArgsParser` for this please?

Reply:
Fixed.